packages/builder/src/builder.js

```diff
@@ -349,7 +349,7 @@ export default class Builder {
           }
           return
         }
-        if (!templateVars.layouts[name] && /\.vue$/.test(file)) {
+        if (!templateVars.layouts[name] || /\.vue$/.test(file)) {
```

Comment:
Can we sort [`layoutFiles`](https://github.com/nuxt/nuxt.js/blob/fc6fb7a75719fdb6b9c5f4fb375d9db340f26ff4/packages/builder/src/builder.js#L339) here to give `.vue` a higher priority or at least add a comment above this line to explain the reason of check? It seems a hidden workaround like this.

Reply:
I think comment is better, since sorting may introduce extra overhead

Reply:
As long as we make it *understandable* to ppl who read the code, why not ☺️